### PR TITLE
Updated unit test for bcsc redirect page

### PR DESCRIPTION
**Pull request by sivakaruna**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Updated unit test for bcsc redirect page

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Unit test

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## For bcgov contributors:

this PR fixes jira ticket: **put the jira ticket # here**
SPDBCSC-483

### CODE REVIEW

File: src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js [hunk 1]
```diff
@@ -1,24 +1,93 @@
...
+    axios.get.mockResolvedValueOnce(axiosCall);
+    const { container } = render(
+      <MemoryRouter>
+        <BcscRedirect page={page} />
+      </MemoryRouter>
+    );
+    await wait(() => {});
+    fireEvent.click(getByText(container, "Login with a BC Services Card"));
+    expect(saveOrg).toHaveBeenCalled();
+    expect(window.open).toHaveBeenCalled();
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js]: can we assert that it is called with specific url?

File: src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js [hunk 2]
```diff
@@ -1,24 +1,93 @@
...
+  });
+
+  test("Validate Redirect to Home when unauthorized", async () => {
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js]: based on the jwt you are generating, this test isnt actually for unauthorized, it is for inadequate actions performed

Comment (sivakaruna) [reply src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js]: The test is for an if condition that checks for authentication and for actions performed. I think we could leave it as unauthorized as a blanket to cover unauthenticated and not allowed to perform a function as it all satisfies the same condition clause.

File: src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js (hunk 1 above)
Comment (sivakaruna) [reply src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js]: Updated

**Pull request merged**